### PR TITLE
Added succes message to store_admin_user_handler

### PR DESCRIPTION
**Pull request by subhajit20**
Added success message to `store_admin_user_handler.rs`
issue - #33

### CODE REVIEW

File: src/api/admin_user/handlers/admin_user_table_handler.rs [hunk 1]
```diff
@@ -47,4 +47,5 @@ pub async fn admin_user_table_handler(
 pub struct AdminUserTableViewModel {
     pub logged_in_user: AdminUserModel,
     pub admin_user_pagination: Pagination,
+    pub success_message: String
```
Comment (indpurvesh) [inline src/api/admin_user/handlers/admin_user_table_handler.rs:50]: I am guessing somewhere around line no #35

     let success_message = session
        .get("success_message")
        .unwrap_or(String::from(""));
    session.remove("success_message");

Comment (indpurvesh) [inline src/api/admin_user/handlers/admin_user_table_handler.rs:50]: On line no #17 you have to add `mut` keyword

    pub async fn admin_user_table_handler(
        mut session: AvoRedSession,
        Query(query_param): Query<AdminUserTableQuery>,
        state: State<Arc<AvoRedState>>,
      ) -> Result<impl IntoResponse> {

**Comment by indpurvesh** [comment]
@subhajit20 Looks like this changes will work. 

But you have to do same thing for Update, delete admin-user/role pages too. 

If you can completed those ones too that would be great.

**Comment by indpurvesh** [comment]
Result of your work. Good
Screenshot 2023-09-11 at 8 19 45 AM

**Comment by indpurvesh** [comment]
You can join slack. `https://join.slack.com/t/avoredrustcms/shared_invite/zt-22031l11y-EYp3a3oWVVFaZ8WCWZAkJQ`

**Comment by subhajit20** [comment]
Joined.

**Comment by subhajit20** [comment]
@indpurvesh will I do the rest in this same branch only or make separate branchs for each solution?

**Comment by indpurvesh** [comment]
You can do it in same it won’t matter

**Pull request merged**